City-of-Helsinki/drupal-asuntotuotanto
# SATT-98: Add checks if the price is zero

## Description
Add check for zero prices

## How to test
Get the dump from production.
`make fresh`
- Index both "Apartment" and "Apartment listing": https://asuntotuotanto.docker.so/fi/admin/config/search/search-api
- Go to https://asuntotuotanto.docker.so/fi/asuntohaku/hitas/asunto-oy-helsingin-priki-johanna-aallonhalkoja-11
- Confirm you are able to see the "Velaton hinta" in the desktop and mobile view and it is not 0.00€
- Click on an apartment and confirm the prices look right
  - On the apartment page confirm that there is no "Asumisoikeusmaksu" visible
- Check also other hitas apartments and confirm that the data looks right

- Check on a couple of haso apartments as well and confirm they still look ok. https://asuntotuotanto.docker.so/fi/asuntohaku/haso
  - Remember to also check the mobile view
